src/components/connection-modal/auto-scanning-step.jsx

```diff
@@ -21,8 +24,13 @@ const PHASES = keyMirror({
     notfound: null
 });
 
-const AutoScanningStep = props => (
-    <Box className={styles.body}>
+const AutoScanningStep = props => {
+    // Offer to update both during scan and after a failed scan, as long there's an update function.
```

Comment:
I didn't see an offer to update during the scan -- should I have?

Or do you mean, offer to update after a successful scan or a failed scan?

<img width="520" alt="image" src="https://github.com/scratchfoundation/scratch-gui/assets/3431616/5a3f03df-d438-488f-97a9-1e56b70060c6">

Reply:
The `AutoScanningStep` and the `ScanningStep` work a little differently for this. The `ScanningStep` makes the offer only after the scan is done, so you can update if the results don't include the one you're looking for. The `AutoScanningStep` will automatically connect to anything it finds, so using the same strategy there would mean that if there's another micro:bit nearby, you'll never get the opportunity to update yours. Because of that, it offers the update before the scan is done.

Reply:
Ah, that makes sense, thanks!